pkg/sbproxy/reconcile_brokers.go

```diff
@@ -193,6 +219,17 @@ func logBroker(broker *platform.ServiceBroker) logrus.Fields {
 	}
 }
 
+func logService(service osbc.Service) logrus.Fields {
+	return logrus.Fields{
+		"service_guid": service.ID,
+		"service_name": service.Name,
+	}
+}
+
+func emptyContext() json.RawMessage {
```

Comment:
This can be a const

Reply:
does not work (not-constant-initializer)